apps/paf/fields/index.js

```diff
@@ -697,7 +697,7 @@ module.exports = {
       'male',
       'female',
       'other',
-      'gender-unknown'
+      'prefer not to say'
```

Comment:
missing the`-`

Reply:
The option would also have to be changed for the other two fields that have been changed to profer-not-to-say

Reply:
Changed in all the three places to prefer-not-to-say